Solutions/DigitalGuardianDLP/Hunting Queries/DigitalGuardianRareDestinationPorts.yaml

```diff
@@ -0,0 +1,24 @@
+id: 82cba92e-fe2f-4bba-9b46-647040b24090
+name: Digital Guardian - Rare destination ports
+description: |
+  'Query searches for rare destination ports.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: DigitalGuardianDLP
+    dataTypes:
+      - DigitalGuardianDLPEvent
+tactics:
+  - Exfiltration
+relevantTechniques:
+  - T1048
+query: |
+  DigitalGuardianDLPEvent
+  | where TimeGenerated > ago(24h)
+  | summarize count() by DstIpAddr, DstPortNumber
+  | order by count_ asc

```

Comment:
It seems this query will list all the ports. How are we determining which are rare. Wondering if we wanted to use
 top 5 or top 10 here.

Reply:
fixed